packages/app/pages/dev.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { ReactElement } from "react";
+import { API } from "@template/api-client";
+import Link from "next/link";
+import styled from "styled-components";
+import { Button, Divider } from "antd";
+import { GetStaticProps } from "next";
+import { PROD_URL } from "@template/common";
+import DefaultErrorPage from "next/error";
+
+const Container = styled.div`
+  width: auto;
+  height: auto;
+  @media (max-width: 768px) {
+    margin: 32px 24px;
+  }
+`;
+
+const LoginContainer = styled.div`
+  margin-left: auto;
+  margin-right: auto;
+  text-align: center;
+
+  padding-top: 100px;
+`;
+
+const SeedingContainer = styled.div`
+  margin-left: auto;
+  margin-right: auto;
+  text-align: center;
+
+  padding-top: 20px;
+`;
+
+const InputBoxContainer = styled.div`
+  margin: auto;
+  width: 500px;
+  height: 64px;
+  left: 570px;
+  top: 261px;
+`;
+
+const PageHeader = styled.div`
+  display: flex;
+  justify-content: center;
+  margin-top: 40px;
+`;
+
+export const getStaticProps: GetStaticProps = async () => {
+  const hidePage = (PROD_URL === process.env.DOMAIN);
```

Comment:
@dajinchu will the `DOMAIN` env be set on the production server?

Reply:
yes, but it's not available in the `packages/app` project... I think I'm going to redo how we handle environment variables